tests/test_workflow.py

```diff
@@ -40,3 +41,22 @@ def test_backtrack():
     numpy.testing.assert_allclose(
         expected_output["e_track"].values, output["e_track"].values
     )
+
+
+def test_visualize():
+    runner = CliRunner()
+    result = runner.invoke(
+        cli, ["visualize", "output", "tests/test_data/test_config.yaml"]
+    )
+    assert result.exit_code == 0
+    output_path = Path("tests/tmp/output_data/figures/cumulative_sources.png")
+    assert output_path.exists()
+    expected_output = Path("tests/test_data/verify_output/cumulative_sources.png")
+    stdout = matplotlib.testing.compare.compare_images(
+        output_path, expected_output, tol=5
+    )
+    if stdout:
+        raise ValueError(
+            "Output figures are different! Please check the difference in the produced"
+            " `diff` figure and verify your results."
```

Comment:
Can you please use the full paths to the diff file? And also it would be really nice to say something like "If you want to keep the new results, you can update the reference figure with `cp tests/tmp/output_data/figures/cumulative_sources.png tests/test_data/verify_output/cumulative_sources.png` and commit the changes.

Reply:
Nice suggestion! Implemented!